src/presets.js

```diff
@@ -1,10 +1,12 @@
-export const presets = {
+import { noop } from './helpers';
+
+export const presets = Object.freeze({
     /**
      * Log method to gain some insights about route navigation and data storing
      * @default
      * @type {Function}
      */
-    log: window.console.log,
+    log: typeof window.console === 'function' ? window.console.log.bind(console) : noop,
```

Comment:
Why the precaution?

Reply:
because the `console` object is a bitch http://tech.genericwhite.com/why-use-console-dot-log-dot-bind-console